Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,203 @@
+
+**_Constraints:_**
+
+_Flywheel Speed:_
+
+•	The minimum flywheel speed required for the resistance system to operate as designed is 9 mph (or 2501.6 flywheel RPM). At flywheel speeds less than 2501.6 RPM, the resistance system will be unable to reach the maximum torque specification, but it will be controlled to provide resistance in as much as possible given the decreasing braking torque with decreasing angular velocity. The value of 9 mph is based on the American Council on Exercise's recommendation regarding the minimum speed for moderate exercising [2].
+
+•	The maximum speed at which the resistance system will operate is pedal 118 rpm (or 6190.3 Flywheel RPM). The maximum speed is determined based on a cyclist traveling at an elite level [3]. When the flywheel is spinning faster than 6190.3 RPM, the resistance system will minimize braking torque to the flywheel to avoid any injury to the rider.
+
+_Flywheel Weight:_
+
+The flywheel construct is required to weigh 20 pounds. The specified flywheel weight is based on data provided by the bike academy that states many indoor cyclists avoid exercise bikes with flywheels over 20 pounds due to the heavier flywheel making it more difficult to generate a desirable momentum [4].
+
+_Resistance States:_
+
+Based on a study conducted by Romagnoli and Piacentini [5], it was found that test subjects could accurately perceive differences in exercise velocity, and in turn resistance changes, approximately 80% of the time, meaning they were inaccurate 20% of the time. However, there is no published documentation that has determined the exact human threshold a human can sense changes in resistance. Using half of the 20% value, a MATLAB script was created to determine what angles resulted in 10% ± 5% increases in resistance. This resulted in 85 unique states. Because of this, the resistance system shall have no less than 85 states of resistance. 
+
+_Maximum Produced Torque:_
```

Comment:
Probably not a relevant citation. What direction was that force applied? A force constraint based on the force to break a leg is not really relevant as a first line of thought. This should be primarily based on the resistance needed to accomplish the objective. Then, you can add mitigations to avoid these concerns.